app/main.ipynb

```diff
@@ -275,17 +960,501 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "outputs": [],
    "source": [
-    "# write your code for task 9 here"
+    "adults_encoded = pd.get_dummies(adults)\n",
+    "adults_encoded.corr()"
```

Comment:
what about `Understand connected fields in it and print highlight thier connection.`?